packages/components/src/focal-point-picker/grid.js

```diff
@@ -0,0 +1,79 @@
+/**
+ * WordPress dependencies
+ */
+import { useRef, useEffect, useState } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import {
+	GridView,
+	GridLineX,
+	GridLineY,
+} from './styles/focal-point-picker-style';
+
+export default function FocalPointPickerGrid( {
+	bounds = {},
```

Comment:
Do you think bounds is better than height/width? I'm mainly thinking in terms of consistency or should it be `dimensions` like the prop for `FocalPointPicker`?